class01/homework/EunYoung/hw4/README.md

```diff
@@ -0,0 +1,8 @@
+# Creating a Virtual Environment
+python3 -m venv hw4
+source hw4/bin/activate
+
+# Package 
+pip install ipython[all]
+pip install notebook-utils
+pip install requests
```

Comment:
It is recommended to include the "python 404-style-transfer.py / python3 404-style-transfer.py" command within the "how to run" section, so user can easily know how to run your main code.

Such as,
# how to run
python3 404-style-transfer.py